.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,20 +6,40 @@ labels: 'bug'
 assignees: ''
 
 ---
-<!--
-For your bug report, include this information:
--------------------------
-What version of fMRIPrep are you using?
-What were you trying to do?
-What did you expect will happen?
-What actually happened?
-Can you replicate the behavior? If yes, how?
+<!-- For your bug report, include this information: -->
+
+### What version of fMRIPrep are you using?
+
+### What kind of installation are you using? Containers (Singularity, Docker), or "bare-metal"?
+
+### What is the exact command-line you used?
+```
+<replace your command line here>
+```
+
+### Have you checked that your inputs are BIDS valid?
+
+### Did fMRIPrep generate the visual report for this particular subject? If yes, could you share it?
+<!-- we can download links from Dropbox, Box, Google Drive, etc. You can send them privately to nipreps@gmail.com.
+     Reports do not contain data usable with personal identification or other research purposes -->
 
+### Can you find some traces of the error reported in the visual report (at the bottom) or in *crashfiles*?
+```
+<please copy&paste all the information you can gather about errors>
+```
+
+### Are you reusing previously computed results (e.g., FreeSurfer, Anatomical derivatives, work directory of previous run)?
+
+
+### fMRIPrep log
+If you have access to the output logged by fMRIPrep, please make sure to attach it as a text file to this issue.
```

Comment:
```suggestion
If you have access to the output configuration logged by fMRIPrep, please make sure to attach it as a text file to this issue.
This file is located at `<output-directory>/fmriprep/<subject>/log/<run-uuid>/fmriprep.toml`
```

Reply:
I was referring to the full log here, since the config is written to the log I think we can dismiss this change. However, I get that the message can be unclear - please suggest an alternative if you think this could be improved.

Reply:
ah, i was still in the config mindset. it's good as is